Checkout/Source/Network/RequestFactory.swift

```diff
@@ -37,18 +37,21 @@ final class RequestFactory: RequestProviding {
 
   enum Request: Equatable {
     case token(tokenRequest: TokenRequest, publicKey: String)
```

Comment:
Not related to PR but should we call the `token` as `cardToken`?

Reply:
good shout, will do since it's an internal type

Reply:
updated this to `cardToken`. We need no doc changes since it's an internal type and this is not a breaking change thereof.